<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>361</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>361</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IMC - Register</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="tb_username_cadastro">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>211</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
color: rgb(150, 150, 150);
padding: 5px;
border: 2px solid rgb(170, 0, 255)
}
QLineEdit:hover{
	border: 2px solid rgb(255, 0, 255)
}
QLineEdit:focus{
	color: rgb(0, 0, 0);
	border: 2px solid rgb(255, 0, 255)
}
</string>
    </property>
    <property name="maxLength">
     <number>25</number>
    </property>
    <property name="placeholderText">
     <string>USER NAME</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_senha_cadastro">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>331</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
color: rgb(150, 150, 150);
padding: 5px;
border: 2px solid rgb(170, 0, 255)
}
QLineEdit:hover{
	border: 2px solid rgb(255, 0, 255)
}
QLineEdit:focus{
	color: rgb(0, 0, 0);
	border: 2px solid rgb(255, 0, 255)
}</string>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>PASSWORD</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_email_cadastro">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>271</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
color: rgb(150, 150, 150);
padding: 5px;
border: 2px solid rgb(170, 0, 255)
}
QLineEdit:hover{
	border: 2px solid rgb(255, 0, 255)
}
QLineEdit:focus{
	color: rgb(0, 0, 0);
	border: 2px solid rgb(255, 0, 255)
}</string>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
    <property name="placeholderText">
     <string>EMAIL</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_confirmasenha_cadastro">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
color: rgb(150, 150, 150);
padding: 5px;
border: 2px solid rgb(170, 0, 255)
}
QLineEdit:hover{
	border: 2px solid rgb(255, 0, 255)
}
QLineEdit:focus{
	color: rgb(0, 0, 0);
	border: 2px solid rgb(255, 0, 255)
}</string>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>CONFIRM PASSWORD</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_enviar_cadastro">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(170, 0, 255);
	color: rgb(255, 255, 255);
	border-radius: 5px;
}
QPushButton:hover {
	color:  rgb(188, 188, 188);
	background-color: rgb(247, 10, 255);
}
QPushButton:pressed {
	color:  rgb(188, 188, 188);
	background-color: rgb(170, 0, 255);
}</string>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background: none;
	color: rgb(170, 0, 255);
	padding: 2px
}</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_nome_cadastro">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
color: rgb(150, 150, 150);
padding: 5px;
border: 2px solid rgb(170, 0, 255)
}
QLineEdit:hover{
	border: 2px solid rgb(255, 0, 255)
}
QLineEdit:focus{
	color: rgb(0, 0, 0);
	border: 2px solid rgb(255, 0, 255)
}
</string>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
    <property name="placeholderText">
     <string>NAME</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_erro_registro">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>321</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
border: 2px solid rgb(85, 0, 0);</string>
    </property>
    <property name="text">
     <string>text</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
</string>
    </property>
    <property name="text">
     <string>ERROR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_lbl_erro_registro">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(170, 0, 255);
	color: rgb(255, 255, 255);
	border-radius: 5px;
}
QPushButton:hover {
	color:  rgb(188, 188, 188);
	background-color: rgb(247, 10, 255);
}
QPushButton:pressed {
	color:  rgb(188, 188, 188);
	background-color: rgb(170, 0, 255);
}</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tb_nome_cadastro</tabstop>
  <tabstop>tb_username_cadastro</tabstop>
  <tabstop>tb_email_cadastro</tabstop>
  <tabstop>tb_senha_cadastro</tabstop>
  <tabstop>tb_confirmasenha_cadastro</tabstop>
  <tabstop>btn_enviar_cadastro</tabstop>
  <tabstop>btn_lbl_erro_registro</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
